phoenix-core/src/it/java/org/apache/phoenix/end2end/ParameterizedIndexUpgradeToolIT.java

```diff
@@ -346,6 +358,26 @@ public void testDryRunAndFailures() throws Exception {
         }
     }
 
+    @Test
+    public void verifyViewAndViewIndexes() throws SQLException {
+        ResultSet rs = conn.createStatement().executeQuery(VIEWS_QUERY);
+        int countViews = -1, indexCount = -1;
+        while(rs.next()) {
```

Comment:
nit: `if (rs.next())` is better? We technically can't go there more than 1 once, since we just `SELECT COUNT`